angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.component.ts

```diff
@@ -351,11 +42,7 @@ export class SprkPromoComponent {
    * @ignore
    */
   getClasses(): string {
-    const classArray: string[] = [
-      'sprk-c-Promo',
-      'sprk-o-Stack',
-      'sprk-o-Stack--split@s',
```

Comment:
I moved sprk-o-Stack--split@s to the outside of the component so that the user can choose when to have the promo split from vertical to side to side based on their content.